packages/lesmis/src/components/MiniMap.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  MapContainer,
+  TileLayer,
+  InversePolygonMask,
+  IconMarker,
+} from '@tupaia/ui-components/lib/map';
+import { TILE_SETS, RED } from '../constants';
+import { useEntityData } from '../api';
+
+const TILE_SET_URL = TILE_SETS.find(t => t.key === 'satellite').url;
+
+const Map = styled(MapContainer)`
+  width: 510px;
+  height: 370px;
+`;
+
+export const MiniMap = ({ entityCode, theme }) => {
+  const { data: entityData, isFetching: isLoadingEntityData } = useEntityData(entityCode);
```

Comment:
Is there a reason why you went for `isFetching` rather than `isLoading`. I would have chosen isLoading because we only really want to show null if there is absolutely not data yet while between fetches (which could happen for instance when clicking breadcrumbs) it might be nicer to keep the map in place and just change the data (I think that would result in a nice map animation)

Reply:
Sorry this was just me not knowing my way around react-query! That's actually quite a cool distinction